lib/ansible/modules/cloud/amazon/elasticache.py

```diff
@@ -128,6 +135,25 @@
     name: "test-please-delete"
     state: rebooted
 
+# Create cache cluster with Tags
+- elasticache:
+    name: "test-please-delete"
+    state: present
+    engine: redis
+    tags: [
```

Comment:
This should use standard ansible tag form

```
tags:
  test: testa
  testb: testc
  testd: testf
```

You'll want to use `ansible_dict_to_boto3_tag_list` for the conversion